share/src/main/resources/alfresco/site-webscripts/org/alfresco/components/guest/login.get.html.ftl

```diff
@@ -30,7 +30,23 @@
          <div class="error">${msg("message.loginautherror")}</div>
          <#else>
          <script type="text/javascript">//<![CDATA[
-            document.cookie = "_alfTest=_alfTest; Path=/;";
+            <#assign cookieHeadersConfig = config.scoped["COOKIES"] />
+            <#if cookieHeadersConfig?? && (cookieHeadersConfig.secure.getValue() == "true" || cookieHeadersConfig.secure.getValue() == "false")>
+               Alfresco.constants.secureCookie = ${cookieHeadersConfig.secure.getValue()};
+               Alfresco.constants.sameSite = "${cookieHeadersConfig.sameSite.getValue()}";
```

Comment:
about the missing IF on sanitize sameSite variable, browsers are automatically switching to "None".